src/goom_config.h

```diff
@@ -2,11 +2,11 @@
 #define GOOM_CONFIG
 
 #ifdef MID
-#define GOOM_TEXTURE_WIDTH  256
-#define GOOM_TEXTURE_HEIGHT 256
-#else  // MID
 #define GOOM_TEXTURE_WIDTH  512
-#define GOOM_TEXTURE_HEIGHT 512
+#define GOOM_TEXTURE_HEIGHT 256
+#else // MID
+#define GOOM_TEXTURE_WIDTH  1280
+#define GOOM_TEXTURE_HEIGHT 720
 #endif // MID
```

Comment:
Have replaced it, for `#ifdef MID` is it still the power of two.
The other use the standard screen resolution. Crazy was here during test that 512x512 and 1280x720 has nearly no difference in CPU usage. Still high ;-) as I've known it from GOOM for years.

Has looked in the source code, there is nothing described about best resolution and on his Test app they use 640x480 (http://goom.cvs.sourceforge.net/viewvc/goom/goom/test/testapp.c?view=markup)